examples/helper-script-gradle/build.gradle

```diff
@@ -0,0 +1,57 @@
+plugins {
+    // Apply the groovy plugin to add support for Groovy
+    id 'groovy'
+    // Apply Gradle JPI plugin to simplify dependency management
+    id 'org.jenkins-ci.jpi' version '0.39.0-rc.7'
+}
+
+group 'com.example'
+version 'O.1-SNAPSHOT'
+
+allprojects {
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+}
+
+sourceSets {
+    main {
+        groovy {
+            srcDirs = ['src', 'vars']
+        }
+    }
+    test {
+        groovy {
+            srcDirs = ['test']
+        }
+    }
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+    jcenter()
+    maven { url 'https://repo.jenkins-ci.org/releases/' }
+}
+
+dependencies {
+    implementation 'org.codehaus.groovy:groovy-all:2.5.10'
+
+    testImplementation 'com.homeaway.devtools.jenkins:jenkins-spock:2.1.2'
+    testImplementation 'ch.qos.logback:logback-core:1.2.3'
+    testImplementation 'ch.qos.logback:logback-classic:1.2.3'
+    testImplementation 'javax.servlet:javax.servlet-api:4.0.1'
+    testImplementation 'org.jenkins-ci.main:jenkins-core:2.190.2', {
+        exclude group: 'com.ibm.icu', module: 'icu4j'
+    }
+    testImplementation 'org.jenkins-ci.plugins.workflow:workflow-cps:2.77'
+    // provides isUnix() step
+    testImplementation 'org.jenkins-ci.plugins.workflow:workflow-basic-steps:2.16'
+    // provides sh() step
+    testImplementation 'org.jenkins-ci.plugins.workflow:workflow-durable-task-step:2.35'
+    // provides sshagent() step
+    testImplementation 'org.jenkins-ci.plugins:ssh-agent:1.16'
+
+    // plugin dependencies
+    testImplementation 'com.cloudbees:groovy-cps:1.31'  // required by workflow-cps plugin
```

Comment:
Am I correct in intuiting that the comment

`// required by workflow-cps plugin`

combined with the [presence of `groovy-cps` as a `<dependency>` of the `workflow-cps` plugin](https://github.com/jenkinsci/workflow-cps-plugin/blob/master/pom.xml#L177-L187) means that transitive dependencies are not resolved by this `build.gradle`?

Reply:
Took out dependency for `groovy-cps`.